crates/bevy_ui/src/render/mod.rs

```diff
@@ -164,7 +164,21 @@ pub struct ExtractedUiNode {
 
 #[derive(Resource, Default)]
 pub struct ExtractedUiNodes {
-    pub uinodes: Vec<ExtractedUiNode>,
+    pub uinodes: Vec<Vec<ExtractedUiNode>>,
```

Comment:
Since the number of extract functions is already known, could this be replaced by a `[Vec<ExtractedUiNode>; 4]`?

Reply:
Needs to be a Vec I think so users still add their own extraction functions to the schedule.

Reply:
A yeah, then it can't be optimized this way. I see already a few crates relying on `ExtractedUiNodes` https://github.com/search?q=ExtractedUiNodes&type=code